src/concurrency.h

```diff
@@ -207,7 +208,7 @@ class single_consumer_frame_queue
 
     bool enqueue( T && item )
     {
-        if( item.is_blocking() )
```

Comment:
How was it working before?
Why was this change needed?

Reply:
Before there was an `is_blocking` function inside the `frame_holder` to make this work... I removed it